src/protocols/ab/tag.h

```diff
@@ -113,6 +113,7 @@ struct ab_tag_t {
     int offset;
 
     int allow_packing;
+    int supports_fragmented_read;
```

Comment:
I probably should make these bit fields.  An int is a lot of wasted space.

Reply:
Haha, yes that might be nice, I tried to make them bools but then realized that the Booleans dtype wasn't defined and didn't want to do that myself.